src/Catalyst.Modules.Server.Blazor/BlazorServerModule.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Diagnostics;
+using Autofac;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Module = Autofac.Module;
+
+namespace Catalyst.Modules.Server.Blazor
+{
+    public class BlazorServerModule : Module
+    {
+        public static void Main(string[] args) { }
+
+        private IHostBuilder _hostBuilder;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_hostBuilder' and declare it as a local variable in the relevant methods.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4594394)